### PR TITLE
added ppl-1.1

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

File: Library/Formula/ppl.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Ppl < Formula
+  desc "Parma Polyhedra Library"
+  homepage "http://bugseng.com/products/ppl"
+  url "ftp://ftp.cs.unipr.it/pub/ppl/releases/1.1/ppl-1.1.tar.xz"
```
Comment (vszakats) [inline Library/Formula/ppl.rb]: http is preferred over ftp, I'd suggest to use this `url`:
http://bugseng.com/products/ppl/download/ftp/releases/1.1/ppl-1.1.tar.xz

File: Library/Formula/ppl.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  def uninstall
+    system "make", "uninstall"
+  end
+
+  test do
+    system "make", "check"
```
Comment (DomT4) [inline Library/Formula/ppl.rb]: This won't work. The `testpath` has no access to the `buildpath`, and vice versa.

File: Library/Formula/ppl.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class Ppl < Formula
+  desc "Parma Polyhedra Library"
+  homepage "http://bugseng.com/products/ppl"
+  url "ftp://ftp.cs.unipr.it/pub/ppl/releases/1.1/ppl-1.1.tar.xz"
+  version "1.1"
```
Comment (DomT4) [inline Library/Formula/ppl.rb]: URL is simple enough that `brew` can understand the version. Can scrap the manual `version` line.

File: Library/Formula/ppl.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  def uninstall
```
Comment (DomT4) [inline Library/Formula/ppl.rb]: We don't need to define this. Our removal code isn't part of formulae.

**Comment by UnixJunkie** [comment]
OK, I took all comments into account

File: Library/Formula/ppl.rb [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  depends_on "gmp"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+end
```
Comment (mikemcquaid) [inline Library/Formula/ppl.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/ppl.rb [hunk 6]
```diff
@@ -0,0 +1,16 @@
+class Ppl < Formula
+  desc "Parma Polyhedra Library"
+  homepage "http://bugseng.com/products/ppl"
```
Comment (mikemcquaid) [inline Library/Formula/ppl.rb]: What's the license?

File: Library/Formula/ppl.rb (hunk 5 above)
Comment (UnixJunkie) [inline Library/Formula/ppl.rb]: I don't know how to do this: you would have to go where the software was installed and run 'make check'.

**Comment by UnixJunkie** [comment]
I added the license info in the desc field

File: Library/Formula/ppl.rb (hunk 5 above)
Comment (mikemcquaid) [inline Library/Formula/ppl.rb]: Please look at the referenced formulae.

**Comment by UnixJunkie** [comment]
I just added a test target inspired by the one from tinyxml2.rb

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `ppl 1.1 (new formula)`.

**Pull request closed without merging**